pkg/operator/operator.go

```diff
@@ -131,6 +135,12 @@ type Options struct {
 	CollectorHTTPClient *http.Client
 }
 
+type getCertificateMethod interface {
```

Comment:
nit: I'm half-tempted to say we should just use `certUpdater` since the API is exactly the same.

Reply:
I didn't want to have `certUpdater` be an exported type, because a `certupdater.CertUpdater{}` would not be useful. Also, it was compatible with certwatcher until I added the GetCA method.